src/worker.cc

```diff
@@ -806,10 +806,17 @@ void from_json(rapidjson::Document& doc, Options::Action action, Api& api) {
       options.set_shape_format(polyline5);
     } else if (*shape_format == "geojson") {
       options.set_shape_format(geojson);
+    } else if (*shape_format == "none") {
```

Comment:
oh i didnt notice this in the json api it is still called `none`

Reply:
hm that was the case since the beginning. but you're right, it's unintuitive to have them differ in PBF/JSON formats, I'll change it